apps/web/src/services/my-page/MyPageLNB.vue

```diff
@@ -52,17 +52,18 @@ export default defineComponent({
         const state = reactive({
             isDomainOwner: computed(() => store.getters['user/isDomainAdmin']),
             hasPermission: computed(() => store.getters['user/hasPermission']),
-            userType: computed(() => store.state.user.backend) as unknown as string,
             userName: computed(() => store.state.user.name),
             email: computed(() => store.state.user.email),
             userId: computed(() => store.state.user.userId),
-            icon: computed(() => {
-                if (state.isDomainOwner) return 'img_avatar_root-account';
-                return 'img_avatar_user';
+            userRoleType: computed(() => store.state.user.roleType),
+            icon: computed<string>(() => {
+                if (store.getters['user/isSystemAdmin']) return 'img_avatar_system-admin';
+                if (store.getters['user/isDomainAdmin']) return 'img_avatar_admin';
+                return 'img_avatar_no-role';
```

Comment:
Suggestion: I think `userRoleType` is unnecessary, or how about using `userRoleType` in `icon` state